playwright.config.ts

```diff
@@ -73,5 +73,6 @@ export default defineConfig({
     command: process.env.CI ? 'pnpm start:static' : 'pnpm run dev',
     url: 'http://localhost:9090',
     reuseExistingServer: !process.env.CI,
+    timeout: 2 * 60 * 1000,
```

Comment:
Is it necessary to add such a long timeout?

Reply:
The default is 1 minute and it failed to build it on my computer. I can remove if it's not necessary.

Reply:
OK. Then let's keep it.